_pages/en_US/installing-boot9strap-(menuhax).txt

```diff
@@ -0,0 +1,89 @@
+
+Info about uloader [here](https://github.com/zoogie/uloader), info about menuhax67 [here](https://github.com/zoogie/menuhax67).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+You should only be able to get to this page if you are running version 11.15.0 or 11.14.0. If you are on any firmware above 11.15.0, uloader will fail. 
+{: .notice--warning}
+
+### What You Need
+
+* The latest release of [SafeB9SInstaller](https://github.com/d0k3/SafeB9SInstaller/releases/download/v0.0.7/SafeB9SInstaller-20170605-122940.zip) (direct download)
+* The latest release of [boot9strap](https://github.com/SciresM/boot9strap/releases/download/1.4/boot9strap-1.4.zip) (direct download)
+* The latest release of [Luma3DS](https://github.com/LumaTeam/Luma3DS/releases/latest) (the Luma3DS `.zip` file)
+* The latest release of [BannerBomb3](https://github.com/zoogie/Bannerbomb3/releases/latest) (the RELEASE `.zip` file)
+* The latest release of [uloader](https://github.com/zoogie/uloader/releases/latest) (the `.3dsx` file)
+
+#### Section I - Prep Work
+
+1. Power off your console
+1. Insert your SD card into your computer
+1. Copy `boot.firm` and `boot.3dsx` from the Luma3DS `.zip` to the root of your SD card
+1. Create a folder named `boot9strap` on the root of your SD card
+1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the boot9strap `.zip` to the `/boot9strap/` folder on your SD card
+1. Copy `SafeB9SInstaller.bin` from the SafeB9SInstaller `.zip` to the root of your SD card
+1. Copy `bb3.bin` from the RELEASE `.zip` to the root of your SD card, replacing the current file
+1. Copy `Launcher.dat` from the folder for your model and region inside the RELEASE `.zip` to the root of your SD card
+1. Create a folder named `3ds` on the root of your SD card if it does not already exist
+    + This folder stores homebrew applications and data; it is different from the `Nintendo 3DS` folder that the console automatically generates
+1. Copy `uloader.3dsx` to the `/3ds/` folder on your SD card
+1. Reinsert your SD card into your console
+
+#### Section II - BannerBomb3
+
+In this section, you will trigger the BannerBomb3 exploit using the DSiWare Management menu and copy the resulting file dump to your computer so that you can use it on the next section.
+
+1. Power on your console
+1. Launch System Settings on your console
+1. Navigate to `Data Management` -> `DSiWare` -> `SD Card` ([image](/images/screenshots/bb3/dsiware-management.png))
+  + Your console should show the BB3 multihax menu
+1. Use the D-Pad to navigate and press the (A) button to select "Install *hax"
+  + Your console will automatically reboot
+
+#### Section III - menuhax67
+
+In this section you will use the menuhax67 exploit installed earlier to launch uloader, which will load the SafeB9SInstaller.
+
+1. Tap on the small HOME Menu settings icon in the top left of the bottom screen
```

Comment:
as opposed to the large home menu settings icon in the top left of the bottom screen
(we might just want to display what it looks like, similar to what we do with DLP?)